src/cli/log.rs

```diff
@@ -65,7 +65,16 @@ where
     S: Subscriber + for<'span> LookupSpan<'span>,
 {
     let maybe_rust_log_directives = process.var_os("RUST_LOG").clone();
-    let has_ansi = process.stderr().is_a_tty();
+    let has_ansi = {
```

Comment:
Can we avoid this nested scope in favor of just assigning `env_override` the value of the `match`?